client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -672,13 +660,10 @@ function getProfessionalEmailUpsellUrl( {
 
 function getDisplayModeParamFromCart(
 	cart: ResponseCart | undefined
-): undefined | { d: 'concierge' | 'traffic-guide' } {
+): undefined | { d: 'concierge' } {
```

Comment:
Not directly related to this PR but I wonder if we still actually support Concierge (aka Quick Start) sessions. I see [some evidence](https://github.com/Automattic/wp-calypso/pull/58867) that they have been deprecated and maybe stopped? If so, we might be able to remove this whole function. cc @joelwills @obenland @wynbarne

Reply:
Quick Starts (concierge) are no longer available for purchase, but if a customer has paid for one, they can still book one through their account. We do around 5 - 10 a week still from the folks who have previously paid.